cpp/include/cudf/strings/repeat_strings.hpp

```diff
@@ -32,15 +32,15 @@ namespace strings {
  */
 
 /**
- * @brief Repeat the given string scalar by a given number of times.
+ * @brief Repeat the given string scalar by a given number of times
```

Comment:
“Repeat by” sounds odd to me.
```suggestion
 * @brief Repeat the given string scalar a given number of times
```